hw8/decorator.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+
+def decorator(func):
+    def wrapper(*args, **kwargs):
+        t = datetime.now()
+        print(f'====\nFunction call time - {t.strftime("%H:%M:%S")}\n====')
+        func(*args, **kwargs)
```

Comment:
Assign the result of the func call to some variable and return it after the last print to be able to decorate functions that return some value